src/libs/localizer.js

```diff
@@ -30,4 +30,9 @@ const getTargetLocales = (targetLang, fallbackLang = `en`) => {
   return locales[TLang]
 }
 
-module.exports = { getTargetLocales }
+const availableLocales = {
```

Comment:
should we plan to use a master table in db? for convenience .. and possibly better control over the variables of each of the languages.

Reply:
So reason for this is, we don't need a db call for this as it won't ever change unless we add or remove a language

Reply:
understandable